README.md

```diff
@@ -11,3 +13,13 @@ mvn clean spring-boot:run -P local -DskipTests
 Test user credentials are set in `pom.xml`:
 * Username: `student@uni.org`
 * Password: `test`
+
+## Running on App Engine
+
+This project is deployed at https://step90-2020.ue.r.appspot.com/ for a limited amount of time.
+
+## Using Activity Hall
+
+Please refer to these user guide videos to learn how to use our website:
+![](individualuser.webm)
```

Comment:
If these embeds don't work in the README, which they probably don't, you can instead add them to you project web site ([GitHub Pages](https://pages.github.com/)), which I just generated in the [gh-pages](https://github.com/googleinterns/step90-2020/tree/gh-pages) branch.
See it live: http://googleinterns.github.io/step90-2020.
You can commit to the branch directly with no PRs, if you like.